src/components/NodeTotalChart/Tooltip.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, Stack, Typography, styled } from '@mui/material';
+import { CSSProperties, FC, useMemo } from 'react';
+import { TooltipProps } from 'recharts';
+
+const StyledContainer = styled(Box)({
+  backgroundColor: "white",
+  borderRadius: "12px",
+  border: "1px solid #2B528B",
+  boxShadow: "0px 4px 4px 0px #00000026",
+  padding: "8px 14px",
+});
+
+const StyledTitle = styled(Typography)({
+  fontFamily: "Inter",
+  fontWeight: 700,
+  fontSize: "14px",
+  lineHeight: "20px",
+  color: "#346798",
+  textTransform: "capitalize",
+  marginBottom: "5px",
+});
+
+const StyledKeyStack = styled(Stack)({
+  marginTop: "5px",
+});
+
+const StyledColorCode = styled(Box, {
+  shouldForwardProp: (p) => p !== "background"
+})<{ background: CSSProperties["backgroundColor"] }>(({ background }) => ({
+  width: "12px",
+  height: "12px",
+  backgroundColor: background,
+}));
+
+const StyledName = styled(Typography)({
+  fontFamily: "Nunito",
+  fontWeight: 600,
+  fontSize: "10px",
+  textTransform: "uppercase",
+  color: "#383838",
+});
+
+const StyledValue = styled(Box)({
+  fontFamily: "Nunito",
+  fontWeight: 400,
+  fontSize: "15px",
+  color: "#595959",
+  lineHeight: "18px",
+  background: "#EAF1F2",
+  padding: "0 4px",
+});
+
+type Props = {
+  normalized: boolean;
+} & TooltipProps<number, SeriesLabel>;
+
+const Tooltip: FC<Props> = ({ active, payload, label, normalized }: Props) => {
```

Comment:
(Optional) Missing tooltip tail pointing at hovered slice in chart. In design it looks to dynamically change from left to right. Not sure how hard it would be to implement this due to Recharts.

Reply:
It seems possible to implement based on the coordinates of where the tooltip is placed but it's a high-ish effort. Leaving this detail omitted ATM. Will revisit in the future.

Thanks for pointing this out!